install_config/cluster_metrics.adoc

```diff
@@ -720,6 +720,32 @@ Information about scaling cluster metrics capabilities is available in the
 xref:../scaling_performance/scaling_cluster_metrics.adoc#cluster-metrics-scaling-openshift-metrics-pods[Scaling and
 Performance Guide].
 
+[[metrics-logging]]
+== Integration with Aggregated Logging
+
+Hawkular Alerts might be connected to the Aggregated Logging's Elasticsearch to react on log events. By default,
+Hawkular will try to find Elasticsearch on its default place (namespace `logging`, pod `logging-es`) at every boot.
+If the Aggregated Logging is installed after Hawkular, the Hawkular Metrics pod might need to be restarted in order
+to recognize the new Elasticsearch server. The Hawkular boot log provides a clear indication if the integration could
+not be properly configured, with messages like:
+
+> Failed to import the logging certificate into the store. Continuing, but the logging integration might fail.
+
+or
+
+> Could not get the logging secret! Status code: 000. The Hawkular Alerts integration with Logging might not work properly.
+
+This feature is available from
+ifdef::openshift-origin[]
+ version v1.6,
+endif::[]
+ifdef::openshift-enterprise[]
+ version 3.6.0,
+endif::[]
+and can be positively confirmed to be available by checking a log entry similar to this:
```

Comment:
Looks better, but just wanted to note that this is part of the same phrase from the previous lines. With the change, this is how it looks like on screen:

> This feature is available from version v1.6. You can confirm if logging is available by checking the log for an entry like: